src/Elastic.Apm/Libraries/Ben.Demystifier/EnhancedStackFrame.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Ben A Adams. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+
+#nullable enable
+namespace System.Diagnostics
```

Comment:
We should change the namespace here to `Elastic.Apm...`, to avoid a conflict where someone also references Ben.Demystifier as a package

Reply:
Agreed - But I ended up with `Elastic.Apm.Ben.Demystifier` otherwise we'd end up with `Elastic.Apm.System...` which would cause some trouble for types in the `Elastic.Apm` namespace that use types from `System`.